cmd/generic-handlers.go

```diff
@@ -631,6 +631,46 @@ func (f bucketForwardingHandler) ServeHTTP(w http.ResponseWriter, r *http.Reques
 		f.handler.ServeHTTP(w, r)
 		return
```

Comment:
Shouldn't we remove the `guessIsBrowserReq` check?

Reply:
yes, correct - it was brought in after rebase.